### PR TITLE
myfirstpullrequest

### PR DESCRIPTION
**Pull request by getbanan**

### CODE REVIEW

**Review by getbanan — COMMENTED** [review]
udalo sie

File: main.py [hunk 1]
```diff
@@ -81,5 +81,7 @@ def main(stdscr):
         if ord(key) == 27:
             break
 
+            print("hello")
```
Comment (mangunowsky) [inline main.py:84]: What is this?

Comment (getbanan) [reply main.py:84]: I think I want to commit it somehow and it works

Comment (mangunowsky) [reply main.py:84]: wydrukuj("dzień dobry")

Good work my g

**Pull request merged**